h/static/styles/simple-search.scss

```diff
@@ -3,6 +3,9 @@
 
 .simple-search {
   overflow: hidden;
+  position: absolute;
```

Comment:
Please try to find a way to do this layout without this. If it's possible to let the layout flow that's always better than absolute positioning in my mind.